code/modules/uplink/uplink_items_fulp.dm

```diff
@@ -101,10 +101,38 @@
 	item = /obj/item/clothing/shoes/digicombat/noslip
 	restricted_species = list("lizard")
 
+/datum/uplink_item/implants/welding
+	name = "Welding Shield Eyes"
+	desc = "These cybernetic eyes will give you protection to flashing lights. Comes with a free autosurgeon."
+	item = /obj/item/autosurgeon/syndicate/welding_eyes
+	cost = 6
+
 /obj/item/clothing/shoes/digicombat/noslip
 	desc = "Robust combat boots especially for lizardmen. Perfect for walking over piled human corpses. This pair of shoes is specifically designed to prevent slipping on wet surfaces."
 	clothing_flags = NOSLIP
 
+/datum/uplink_item/race_restricted/plasmemehardsuit
+	name = "Plasmaman Syndicate Hardsuit"
+	desc = "This special optimized hardsuit gives the best of both hardsuit modes, while preventing our plasmamen \
+			operatives from setting themselves on fire, due to the nature of our other hardsuits."
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+	exclude_modes = list()
+	cost = 6 // The normal Space Suit costs 4TC, with the Elite hardsuit priced at the double of the Space suit.
+	item = /obj/item/clothing/suit/space/hardsuit/syndiplasmeme
+	restricted_species = list("plasmaman")
+	limited_stock = 1 // No, we're not letting you give your entire nuke team no-slowdown EVA hardsuits. They can buy Elite for 2 more TC, anyways.
```

Comment:
a plasmaman operative could theoretically buy one of these hardsuits for themselves, then buy a plasmaman elite syndicate hardsuit for one of their non-plasmaman buddies

Reply:
correct, which i'd just let pass imo.
if a plasma man is crippling themselves to give another teammate another advantage, then i'd let them